app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,8 @@
+<% provide :title, "Help"%>
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the
```

Comment:
I18n cho toàn bộ text hiển thi ở view nhé

self check file bên dưới luôn

nhớ thay `a` thành `link_to`